modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm-volley/auth/apikeyauth.mustache

```diff
@@ -0,0 +1,16 @@
+     // Api Key auth supports query header and cookie.
+     // Query is supported in the path generation only with a hardcoded value.
+     // TODO: Not sure about cookie auth form
+     // If implementing api key in query parameter use the ^isKeyInHeader property
+
+     val apiKeyAuthHeaderFactoryBuilder = {
+          paramName: String, apiKeyPrefix: String?, apiKey: String? -> {
+              mapOf("$paramName" to
```

Comment:
@alisters could you please replace `mapOf("$paramName" to` with `mapOf(paramName to`?

Reply:
Fixed and this will resolve the issue below, when samples are regenerated